pkg/controller/projects/members/controller_test.go

```diff
@@ -33,15 +33,17 @@ import (
 	"github.com/crossplane-contrib/provider-gitlab/apis/projects/v1alpha1"
 	"github.com/crossplane-contrib/provider-gitlab/pkg/clients/projects"
 	"github.com/crossplane-contrib/provider-gitlab/pkg/clients/projects/fake"
+	"github.com/crossplane-contrib/provider-gitlab/pkg/clients/users"
 )
 
 var (
 	unexpecedItem     resource.Managed
 	errBoom           = errors.New("boom")
-	ID                = 0
-	extName           = strconv.Itoa(ID)
+	ProjectID         = 0
+	extName           = strconv.Itoa(ProjectID)
```

Comment:
Local variables should be lower case
```suggestion
	projectID         = 0
	extName           = strconv.Itoa(projectID)
```